mesonbuild/backend/vs2010backend.py

```diff
@@ -1099,6 +1099,11 @@ def gen_vcxproj(self, target, ofname, guid):
         else:
             ET.SubElement(clconf, 'FavorSizeOrSpeed').text = 'Speed'
         # Note: SuppressStartupBanner is /NOLOGO and is 'true' by default
+        if self.name in('vs2017', 'vs2019'):
```

Comment:
`in` is a keyword, there should be a space after `in`.

Reply:
done